inductiva/utils/format_utils.py

```diff
@@ -235,3 +235,47 @@ def get_tabular_str(tabular_data: Union[Mapping[str, Iterable[Any]],
         table = _table_indenter(table, indentation_level)
 
     return f"\n{table}\n"
+
+
+def currency_formatter(amount, currency="USD"):
+    """Format a currency amount into a human-readable string."""
+    # Define currency-specific information
+    currency_info = {
+        "USD": {
+            "symbol": "$"
+        },
+        "EUR": {
+            "symbol": "€"
+        },
+        "GBP": {
+            "symbol": "£"
+        },
+        # Add more currencies here if needed
+    }
```

Comment:
I've ended up removing this. Since we already have US$ hardcoded there is no point in implementing this so soon. In the future we might get the currency from the back end.